src/plugins/visualizations/public/embeddable/visualize_embeddable.tsx

```diff
@@ -393,31 +394,35 @@ export class VisualizeEmbeddable
         const { error } = this.getOutput();
 
         if (error) {
-          if (isFallbackDataView(this.vis.data.indexPattern)) {
-            render(
-              <VisualizationMissedSavedObjectError
-                viewMode={this.input.viewMode ?? ViewMode.VIEW}
-                renderMode={this.input.renderMode ?? 'view'}
-                savedObjectMeta={{
-                  savedObjectId: this.vis.data.indexPattern.id,
-                  savedObjectType: this.vis.data.savedSearchId
-                    ? 'search'
-                    : DATA_VIEW_SAVED_OBJECT_TYPE,
-                }}
-                application={getApplication()}
-              />,
-              this.domNode
-            );
-          } else {
-            render(<VisualizationError error={error} />, this.domNode);
-          }
+          this.renderError(this.domNode, error);
```

Comment:
Do we still need to call it here? Seems like the base class is calling `renderError` as soon as the output has an error, so we can remove this call, right?